src/jit/emitarm.cpp

```diff
@@ -6853,7 +6883,7 @@ void                emitter::emitDispReg(regNumber reg, emitAttr  attr, bool add
 {
     if (isFloatReg(reg))
     {
-        char *size = attr == EA_8BYTE ? "d" : "s";
+        char *size = (char *)(attr == EA_8BYTE ? "d" : "s");
```

Comment:
I would prefer changing the type to const char* instead of casting the constant strings to non-const.